src/assisted_test_infra/test_infra/utils/utils.py

```diff
@@ -272,6 +274,13 @@ def get_local_assisted_service_url(namespace, service, deploy_target):
 
         raise RuntimeError(f"The parsed url {url} to service {service} in {namespace} namespace was not reachable.]")
 
+def get_assisted_service_url_by_configmap(configmap, namespace):
```

Comment:
what's the difference from ``get_local_assisted_service_url``?

Reply:
Thanks for the catch! I didn't look closely at that function. Removed the configmap function!